nvssclient/Controllers/RecordController.cs

```diff
@@ -155,12 +160,88 @@ public async Task<ActionResult> VoidRecordHandler([FromBody] List<object> textLi
+                uint? blockCount = UInt32.Parse(json["block_count"].ToString());
                 DeathRecord record = new DeathRecord(recordJson.ToString(), true);
+                
                 var message = new DeathRecordVoidMessage(record);
+                message.BlockCount = blockCount;
+                InsertMessageItem(message);
+            } catch (Exception e){
+                Console.WriteLine("Error Handling Record: {0}", e);
+                return BadRequest();
+            }
+            // return HTTP status code 204 (No Content)
+            return NoContent();
+        }
+
+        // POST: Alias Records
+        // Wraps each record in a FHIR Alias message and queues the message to be sent to the NVSS API Server
+        [HttpPost]
+        [Route("aliases")]
+        public async Task<ActionResult> AliasRecordHandler([FromBody] List<object> textList)
+        {               
+            try {
+                foreach (object text in textList)
+                {
+                    // Create a submission message for the record
+                    var dict = JsonConvert.DeserializeObject<Dictionary<string, object>>(text.ToString());
+                    object recordJson = dict["record"];
+                    DeathRecord record = new DeathRecord(recordJson.ToString(), true);
+                    string firstName = dict["alias_decedent_first_name"].ToString();
+                    string lastName = dict["alias_decedent_last_name"].ToString();
+                    string middleName = dict["alias_decedent_middle_name"].ToString();
+                    string suffixName = dict["alias_decedent_name_suffix"].ToString();
+                    string fatherSurname = dict["alias_father_surname"].ToString();
+                    string ssn = dict["alias_social_security_number"].ToString();
+
+                    var message = new DeathRecordAliasMessage(record);
+                    message.AliasDecedentFirstName = firstName;
+                    message.AliasDecedentLastName = lastName;
+                    message.AliasDecedentMiddleName = middleName;
+                    message.AliasDecedentNameSuffix = suffixName;
+                    message.AliasFatherSurname = fatherSurname;
+                    message.AliasSocialSecurityNumber = ssn;
+                    InsertMessageItem(message);
+                }
+            } catch (Exception e){
+                Console.WriteLine("Error Handling Record: {0}", e);
+                return BadRequest();
+            }
+
+            // return HTTP status code 204 (No Content)
+            return NoContent();
+        }
+
+        // POST: Alias Records
+        // Wraps the record in a FHIR Alias message, sets the alias values and queues the message to be sent to the NVSS API Server
+        [HttpPost]
+        [Route("alias")]
+        public async Task<ActionResult> AliasRecordHandler([FromBody] Dictionary<string, object> data)
```

Comment:
This method is declared as asynchronous but does not await anything. Either drop the `async` or use `await` to wait on an asynchronous operation (such as the database).